README.md

```diff
@@ -39,6 +39,24 @@ This document will walk you through getting up and running with the .NET SDK Sam
 1. Setting up build environment
 2. Setting up a vSphere test environment
 
+### Importing, building and running the Project Template 
```

Comment:
Make the topic as "Adding a new sample using a template" and add it after "Running the samples" section. We don't want this to be the first thing people do after cloning the repo and going through quickstart.

Reply:
ok sure